lib/wallet/rpc.js

```diff
@@ -2499,7 +2499,7 @@ class RPC extends RPCBase {
  _validateBatch(args, help, method) {
```

Comment:
To stay consistent with existing create* RPC method behaviour, `_validateBatch` can maybe set default `options.path = true` since all other RPC calls do this:
https://github.com/handshake-org/hsd/blob/3fd74e0dd8f822590cf30a0f143d0766dfd2d9c0/lib/wallet/rpc.js#L2170-L2174

Reply:
added